test/test_linalg.py

```diff
@@ -261,12 +261,13 @@ def select_if_not_empty(t, i):
 
         def check_correctness_scipy(a, b, res, driver, cond):
             # SciPy provides 3 driver options: gelsd, gelss, gelsy
-            if TEST_SCIPY and driver in ('gelsd', 'gelss', 'gelsy'):
-                import scipy.linalg
+            if device == 'cpu':
+                if TEST_SCIPY and driver in ('gelsd', 'gelss', 'gelsy'):
```

Comment:
I was running into this error
```
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.
```

And as I write this comment I should just transfer it to the host.